protocol/scripts/governance/bridge_vesters_set_rewards.py

```diff
@@ -92,9 +94,23 @@ def estimate_block_height(current_block_height, current_block_timestamp, average
     AVG_BLOCK_TIME_FOR_ESTIMATE,
     UPDATE_2_TIME_UTC,
 )
+# Get current time in UTC
+current_utc_time = datetime.now(pytz.utc)
```

Comment:
nit: this means that the proposal should be submitted pretty shortly after this script is used right? let's at least make this clear to the user!

Reply:
Makes sense, will note that a few hours of difference is probably fine since estimate itself is pretty rough (accuracy may be of by ~24 hours or more)